Readme.org

```diff
@@ -124,18 +124,18 @@ return Sex code.
   `(if (< 0 ,call)
        (begin
          (puts ,message)
-         (return ,ret-code))))
+         (return ,ret-code)))))
 
-(pub fn int init ()
+(pub fn init () int
   (check-sdl-return
     (SDL-Init SDL-INIT-VIDEO) "Failed to initialize SDL" 1)
 ...)
 #+end_src
 ->
-#+begin_src c
-(%fun int init ()
+#+begin_src scheme
+(pub fn init () int
   (if (< 0 (SDL_Init SDL_INIT_VIDEO))
-    (%begin (puts "Failed to initialize SDL") (return 1)))
+    (begin (puts "Failed to initialize SDL") (return 1)))
   ...)
 }
```

Comment:
Good stuff, good stuff. But there's a smol momentik left :eye::eye:

Reply:
What of it?